doc/source/timeseries.rst

```diff
@@ -647,6 +647,33 @@ Another example is parameterizing ``YearEnd`` with the specific ending month:
    d + YearEnd()
    d + YearEnd(month=6)
 
+
+.. _timeseries.offsetseries:
+
+Using offsets with ``Series`` / ``DatetimeIndex``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Offsets can be used with either a ``Series`` or ``DatetimeIndex`` to
```

Comment:
provide a link to this section as its pretty similar: http://pandas.pydata.org/pandas-docs/stable/timedeltas.html#operations (feel free to adjust language in either as needed)